mobile/android/app/build_defs.bzl

```diff
@@ -0,0 +1,19 @@
+"""Helper definitions to glob .aar and .jar targets"""
+
+def create_aar_targets(aarfiles):
```

Comment:
Useful extraction

Reply:
Btw idk what it means, some RN codegen magic here. Can you explain pls?

Reply:
They are using bazel, nice